diagnostic.md

```diff
@@ -6,21 +6,25 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The main task to perform in the application router is
+    configuering routes using this.route(''), so the router knows
+    which view-state to show for that given route. The main task in an
+    Ember route is to provide a model hook which returns information
+    similar to the structure of JSON.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.